src/plugins/BrowserStorageFacade.ts

```diff
@@ -0,0 +1,86 @@
+import { EncryptedKey } from "../types";
+
+export interface BrowserStorageConfigParams {
+  prefix?: string;
+  storage: any;
+}
+
+/**
+ * Facade for `BrowserStorageKeyStore` encapsulating the access to the actual
+ * browser storage
+ */
```

Comment:
this file is very similar to `LocalStorageFacade`. This just takes into account browser storage's slightly different API